src/App.js

```diff
@@ -1,20 +1,21 @@
-import React from "react";
-import "./App.css";
-import GameBoard from "./components/GameBoard";
+import React from 'react'
+import './App.css'
+import GameBoard from './components/gameboard'
 
 // App is the top-level component that renders the entire GameBoard
 function App() {
-	return (
-		<div className="App" style={{ textAlign: "center" }}>
```

Comment:
For cleanliness on the review, I would prefer whitespace and linting changes on a separate commit. However, I appreciate you are moving fast and using your default local dev settings, so I will let it pass. Makes it harder for me to review the actual feature changes.